cmd/wallet-server/startcmd/start.go

```diff
@@ -93,6 +93,10 @@ const (
 	keyEDVURLFlagName  = "key-edv-url"
 	keyEDVURLFlagUsage = "Operational key EDV Server URL"
 	keyEDVURLEnvKey    = "HTTP_SERVER_KEY_EDV_URL"
+
+	useRemoteKMSFlagName  = "use-remote-kms"
+	useRemoteKMSFlagUsage = "Use Remote KMS? (y/n)"
+	useRemoteKMSEnvKey    = "USE_REMOTE_KMS"
```

Comment:
Followup: just like we have the concept of "database type" in the configuration, we should have the same for KMS. Its value should be "webkms" in this case.

Reply:
it's just a boolean for now, either local or remote, that's a possible idea though, if we want to match the client kms implementation with the server's implementation. Maybe it's good for a future release.

Reply:
it has been update to kmsType (in a separate PR)